components/raftstore/src/coprocessor/config.rs

```diff
@@ -76,7 +76,11 @@ pub const RAFTSTORE_V2_SPLIT_SIZE: ReadableSize = ReadableSize::gb(10);
 /// Default batch split limit.
 pub const BATCH_SPLIT_LIMIT: u64 = 10;
 
-pub const DEFAULT_BUCKET_SIZE: ReadableSize = ReadableSize::mb(96);
+// A bucket will be split only when its size is larger than 2x of
+// DEFAULT_BUCKET_SIZE So the avg of the actual bucket size is 75MB, which is
+// slightly less than region size We don't use 48MB size because it will enable
+// the automatic bucket under default 96MB region size.
```

Comment:
what you mean automatic bucket? I remember the bucket is not enabled if dynamic region is not set.

Reply:
No, bucket is enabled automatically when bucket size * 2 <= region size, regardless dynamic region or not @SpadeA-Tang 

  if let Ok(()) = res && self.enable_region_bucket.is_none() {
            let useful = self.region_split_size() >= self.region_bucket_size * 2;
            self.enable_region_bucket = Some(useful);
        } else if let Err(e) = res && self.enable_region_bucket() {
            return Err(e);
        }